spec/system/kudo/edit_spec.rb

```diff
@@ -38,6 +39,24 @@
         expect(page).to have_content 'Lorem ipsum dolor sit amet'
         expect(page).to have_content 'consectetur adipiscing elit. Sed dignissim dignissim urna vel ultrices.'
       end
+
+      it 'fails after 5 minutes from kudo\'s creation' do
+        logged_employee = create(:employee)
+        sign_in logged_employee
+        owned_kudo = create(:kudo, giver: logged_employee)
+        visit edit_kudo_path(owned_kudo)
```

Comment:
I suggest using click_on instead.
- first of all the system specs are from the perspective of user - he is not typing in the address bar, he is clicking around
- you will check if your button is working -> if you later change the 'edit' button for some reason this test won;t pick it up